.pre-commit-config.yaml

```
@@ -29,6 +29,11 @@ repos:
       language: python
       types: [python]
       log_file: '.artifacts/flake8.pycodestyle.log'
+-   repo: https://github.com/sirosen/check-jsonschema
+    rev: 0.3.0
+    hooks:
+    -   id: check-github-actions
+    -   id: check-github-workflows
```

Let's please only run these on relevant files.

```suggestion
    -   id: check-github-actions
        files: ^\.github/actions/
        types: [yaml]
    -   id: check-github-workflows
        files: ^\.github/workflows/
        types: [yaml]
```

---

Oh, I dunno if this is default behavior, but still, good to be explicit here.

---

It is the default:
https://github.com/sirosen/check-jsonschema/blob/20092d2bd4935808ba5c40dd60e57e9df9f1ac05/.pre-commit-hooks.yaml#L9-L26
```
- id: check-github-workflows
  name: Validate GitHub Workflows
  description: 'Validate GitHub Workflows against the schema provided by Schemastore'
  entry: check-jsonschema --schemafile "https://json.schemastore.org/github-workflow"
  language: python
  files: ^\.github/workflows/
  types: [yaml]
- id: check-github-actions
  name: Validate GitHub Actions
  description: 'Validate GitHub Actions against the schema provided by Schemastore'
  entry: check-jsonschema --schemafile "https://json.schemastore.org/github-action"
  language: python
  files: >
    (?x)^(
      action.(yml|yaml)|
      \.github/actions/.*
    )$
  types: [yaml]
```